### PR TITLE
fix slip adjust formula

### PR DESCRIPTION
**Pull request by AustinoBombino**
Update: slipAdjustment = (1 - ABS((R a - r A)/((2 r + R) (a + A)))) 
To: slipAdjustment = (1 - ABS((R a - r A)/((r + R) (a + A))))

### CODE REVIEW

**Review by Jedi2002 — APPROVED** [review]
lgtm

File: x/clp/keeper/calculations.go [hunk 1]
```diff
@@ -154,11 +155,11 @@ func CalculatePoolUnits(symbol string, oldPoolUnits, nativeAssetBalance, externa
...
 	if adjustExternalToken {
-		externalAssetAmount = externalAssetAmount.Mul(sdk.NewUintFromBigInt(normalizationFactor.BigInt())) // Convert token which are not E18 to E18 format
-		externalAssetBalance = externalAssetBalance.Mul(sdk.NewUintFromBigInt(normalizationFactor.BigInt()))
+		externalAssetAmount = externalAssetAmount.Mul(sdk.NewUintFromBigInt(normalizationFactor.RoundInt().BigInt())) // Convert token which are not E18 to E18 format
+		externalAssetBalance = externalAssetBalance.Mul(sdk.NewUintFromBigInt(normalizationFactor.RoundInt().BigInt()))
 	} else {
-		nativeAssetAmount = nativeAssetAmount.Mul(sdk.NewUintFromBigInt(normalizationFactor.BigInt()))
-		nativeAssetBalance = nativeAssetBalance.Mul(sdk.NewUintFromBigInt(normalizationFactor.BigInt()))
+		nativeAssetAmount = nativeAssetAmount.Mul(sdk.NewUintFromBigInt(normalizationFactor.RoundInt().BigInt()))
+		nativeAssetBalance = nativeAssetBalance.Mul(sdk.NewUintFromBigInt(normalizationFactor.RoundInt().BigInt()))
```
Comment (AustinoBombino) [inline x/clp/keeper/calculations.go:162]: @Jedi2002 @iverc added .RoundInt() to the calc pool units to match normalization in other functions.

was causing overflow with large numbers without it

**Review by iverc — APPROVED** [review]

File: x/clp/keeper/calculations.go (hunk 1 above)
Comment (iverc) [reply x/clp/keeper/calculations.go:162]: 🚀

**Comment by iverc** [comment]
@AustinoBombino @Jedi2002 it won't pass tests cause code is not changed yet, let me combine it with my changes

**Comment by iverc** [comment]
Included in #845

**Pull request closed without merging**